zookeeper/src/main/java/com/linecorp/armeria/client/zookeeper/StoreType.java

```diff
@@ -0,0 +1,8 @@
+package com.linecorp.armeria.client.zookeeper;
+
+/**
+ * Value store type. As a node value or as a node's all Children.
+ */
+public enum StoreType {
```

Comment:
How about:

- making this an inner type of `ZooKeeperEndpointGroup` and
- renaming this to `Mode`?